java/src/jmri/jmrit/ctc/CTCMain.java

```diff
@@ -1,5 +1,6 @@
 /**
  *  @author Gregory J. Bedlek Copyright (C) 2018, 2019
+ *  Comment to force another CI build
```

Comment:
Not really a problem, but these extra comments tend to accumulate in an untidy way.  Generally people will restart CI once or twice a day, so if you can wait it'll get taken care of.  Or a quick open-and-close of the PR will also restart the CI.